src/Mailgun/Api/Suppressions/Complaint.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Api;
+
+use Mailgun\Assert;
+use Mailgun\Resource\Api\Suppressions\Complaint\CreateResponse;
+use Mailgun\Resource\Api\Suppressions\Complaint\DeleteResponse;
+use Mailgun\Resource\Api\Suppressions\Complaint\IndexResponse;
+use Mailgun\Resource\Api\Suppressions\Complaint\ShowResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class Complaint extends HttpApi
+{
+    use Pagination;
+
+    /**
+     * @param string $domain
+     *
+     * @return IndexResponse
+     */
+    public function index($domain)
+    {
+        Assert::notEmpty($domain);
+        Assert::range($limit, 1, 10000, 'Limit parameter must be between 1 and 10000');
```

Comment:
`$limit` parameter was not requested neither on `index()` method nor on annotations. This should throw a PHP error. Also, I cannot see where you're using it on this call, so do we really need it?